tests/transforms/test_transforms.py

```diff
@@ -215,12 +203,12 @@ def test_extract_patches() -> None:
         'image': torch.randn(size=(b, c, h, w)),
         'mask': torch.randint(low=0, high=2, size=(b, h, w)),
     }
-    train_transforms = transforms.AugmentationSequential(
-        _ExtractPatches(window_size=p), same_on_batch=True, data_keys=['image', 'mask']
-    )
-    output = train_transforms(batch)
-    assert batch['image'].shape == (b * num_patches, c, p, p)
-    assert batch['mask'].shape == (b * num_patches, p, p)
+    train_transforms = ExtractTensorPatches(p, s)
```

Comment:
This should be used inside of kornia's AugmentationSequential so we can know that it works inside the datamodules. Our datamodules don't apply to image and mask in separate calls.